contributors/guide/community-expectations.md

```diff
@@ -16,53 +16,56 @@ that shapes the rest of the project.  It currently covers the expectations
 of conduct that govern all members of the community as well as the expectations
 around code review that govern all active contributors to Kubernetes.
 
-### Code of Conduct
+## Code review
 
-The most important expectation of the Kubernetes community is that all members
-abide by the Kubernetes [community code of conduct](../../governance.md#code-of-conduct).
-Only by respecting each other can we develop a productive, collaborative
-community.
-
-### Code review
-
-As a community we believe in the [value of code review for all contributions](collab.md).
+As a community we believe in the value of code review for all contributions.
 Code review increases both the quality and readability of our codebase, which
 in turn produces high quality software.
 
 However, the code review process can also introduce latency for contributors
 and additional work for reviewers that can frustrate both parties.
 
 Consequently, as a community we expect that all active participants in the
-community will also be active reviewers.
-
-We ask that active contributors to the project participate in the code review process
-in areas where that contributor has expertise. Active
-contributors are considered to be anyone who meets any of the following criteria:
-   * Sent more than two pull requests (PRs) in the previous one month, or more
-   than 20 PRs in the previous year.
-   * Filed more than three issues in the previous month, or more than 30 issues in
-   the previous 12 months.
-   * Commented on more than five pull requests in the previous month, or
-   more than 50 pull requests in the previous 12 months.
-   * Marked any PR as LGTM in the previous month.
-   * Have *collaborator* permissions in the Kubernetes github project.
-
-In addition to these community expectations, any community member who wants to
-be an active reviewer can also add their name to an *active reviewer* file
-(location tbd) which will make them an active reviewer for as long as they
-are included in the file.
-
-#### Expectations of reviewers: Review comments
+community will also be active reviewers. The 
+[community membership](../community-membership.md) outlines the responsibilities
+of the different contributor roles. 
+
+All changes must be code reviewed. For non-maintainers this is obvious, since
+you can't commit anyway. But even for maintainers, we want all changes to get at
+least one review, preferably (for non-trivial changes obligatorily) from someone
+who knows the areas the change touches. For non-trivial changes we may want two
+reviewers. The primary reviewer will make this decision and nominate a second
+reviewer, if needed. Except for trivial changes, PRs should not be committed
+until relevant parties (e.g. owners of the subsystem affected by the PR) have
+had a reasonable chance to look at PR in their local business hours.
+
+Most PRs will find reviewers organically. If a maintainer intends to be the
```

Comment:
This also isn't accurate, as we have bots in most repos to do this.

Reply:
I'll remove all this and we'll just have all the bot docs in one place.